src/main/resources/script/cbms_scriptsV4/V4.02.sql

```diff
@@ -0,0 +1,87 @@
+-- This script will make the changes AS requested in issue #112
+
+-- Adding the Archived column in asset to replace the train from dataset and transfer values
+ALTER TABLE cbms.asset ADD COLUMN archived INT(1) NOT NULL DEFAULT '0';
+UPDATE cbms.asset a, cbms.dataset_asset_assoc daa, cbms.dataset d SET a.archived='1' WHERE 
+a.asset_id = daa.asset_id AND
+daa.dataset_id = d.dataset_id AND
+d.train = '1';
+
+-- Remove all constraints, drop the primary key, remake constraints, remake primary key without dataset
+-- Remove dataset_id column from table trained model
+ALTER TABLE cbms.trained_model ENGINE=InnoDB;
+ALTER TABLE cbms.trained_model DROP FOREIGN KEY fk_table1_dataset1;
+ALTER TABLE cbms.trained_model DROP FOREIGN KEY fk_table1_asset_type1;
+ALTER TABLE cbms.trained_model DROP FOREIGN KEY fk_table1_model1;
+DROP INDEX fk_table1_dataset1_idx ON cbms.trained_model;
+DROP INDEX PRIMARY ON cbms.trained_model;
```

Comment:
I know In doesn't work on MySQL but since we're using MariaDB I don't see the issue. If anyone is using the wrong database it's up to them to correct that and use the chosen one.

Reply:
I agree with that statement. I just wanted to make sure. Nice job, I'll approve then!